tests/specs/tests.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('tests', () => {
                 topic: 'bar',
                 key: 'thekey',
                 value: {data: 'bar'},
-                headers: {eventType: 'test2', source: 'test-service2'},
+                headers: {eventType: 'test2', source: 'test-service2', nullHeader: null},
```

Comment:
let's keep one with null and one without null (for sake of scenarios coverage), what do you think?